masterlist.yaml

```diff
@@ -19541,6 +19665,8 @@ plugins:
     group: *highPriorityGroup
     inc: [ 'Vaults of Skyrim 1.0 NEXUS.esp' ]
     msg:
+      - <<: *compatPatch_JKIPC
+        condition: 'active("JK''s (Blue Palace|Dragonsreach|Mistveil Keep|Palace of the Kings|Understone Keep)\.esp")'
```

Comment:
Since you've already added this message to all of those plugins, they should suffice. It's also a bit misleading as the patch compilation is for JK's Interiors, not PCE.

```suggestion
```